apis/swagger.yml

```diff
@@ -478,9 +476,7 @@ definitions:
         description: |
           The entry point for the container as a string or an array of strings.
           If the array consists of exactly one empty string (`[""]`) then the entry point is reset to system default (i.e., the entry point used by docker when there is no `ENTRYPOINT` instruction in the `Dockerfile`).
-        type:
-          - "array"
-          - "string"
```

Comment:
I don't think it is necessary to add a validation here and change the type definition to fit the swagger 2.0 specification.

The moby project also used a "mixed type definition" in their [yaml](https://github.com/moby/moby/blob/master/api/swagger.yaml#L836), it won't pass the validation, but it can generate the right document

Swagger OAS 2.0  does not support "mixed type" while [OAS 3.0](https://swagger.io/docs/specification/data-models/oneof-anyof-allof-not/) does. For now, the toolchain of OAS 3.0 is not ready, but when it is ready I think we can adapt this yaml to OAS 3.0  and add validation.

Reply:
Thanks for your detailed explanation. I think this information is very important for us.
Could you evaluate this for pouch ? @CodeJuan

Reply:
I can't agree more. We should support `mixed type definition` when the toolcharin of OAS 3.0 is ready.